### ISSUE TITLE
Can install .net8 Error acquiring .net 

### ISSUE THREAD

**Issue by rmarinho** [dotnet/vscode-dotnet-runtime#1815]
### Describe the bug.

Seems when we try to open .NET MAUI repo with VSCode we get some issues with .net install tool, specially when we try to load with .net9 on the system. 

### Please attach log file(s) here if you have any.

```shell
ms-dotnettools.csdevkit requested to download the .NET Runtime.
Downloading .NET version(s) 8.0.5~arm64 ....Error
Failed to download .NET 8.0:
Web Request to https://dot.net/v1/dotnet-install.sh Failed: n[r].split is not a function. Aborting. Stack: TypeError: n[r].split is not a function
	at /Users/ruimarinho/.vscode/extensions/ms-dotnettools.vscode-dotnet-runtime-2.0.5/dist/extension.js:2:753548
	at a (/Users/ruimarinho/.vscode/extensions/ms-dotnettools.vscode-dotnet-runtime-2.0.5/dist/extension.js:2:753596)
	at Function.s [as headerInterpreter] (/Users/ruimarinho/.vscode/extensions/ms-dotnettools.vscode-dotnet-runtime-2.0.5/dist/extension.js:2:698603)
	at n (/Users/ruimarinho/.vscode/extensions/ms-dotnettools.vscode-dotnet-runtime-2.0.5/dist/extension.js:2:702699)
	at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
	at async Pt.request (/Users/ruimarinho/.vscode/extensions/ms-dotnettools.vscode-dotnet-runtime-2.0.5/dist/extension.js:2:748180)
	at Pt.request (/Users/ruimarinho/.vscode/extensions/ms-dotnettools.vscode-dotnet-runtime-2.0.5/dist/extension.js:2:748273)
	at process.processTicksAndRejections (node:internal/process/task_queues:95:5)

Error
Failed to download .NET 8.0:
Web Request to https://dot.net/v1/dotnet-install.sh Failed: n[r].split is not a function. Aborting. Stack: TypeError: n[r].split is not a function
	at /Users/ruimarinho/.vscode/extensions/ms-dotnettools.vscode-dotnet-runtime-2.0.5/dist/extension.js:2:753548
	at a (/Users/ruimarinho/.vscode/extensions/ms-dotnettools.vscode-dotnet-runtime-2.0.5/dist/extension.js:2:753596)
	at Function.s [as headerInterpreter] (/Users/ruimarinho/.vscode/extensions/ms-dotnettools.vscode-dotnet-runtime-2.0.5/dist/extension.js:2:698603)
	at n (/Users/ruimarinho/.vscode/extensions/ms-dotnettools.vscode-dotnet-runtime-2.0.5/dist/extension.js:2:702699)
	at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
	at async Pt.request (/Users/ruimarinho/.vscode/extensions/ms-dotnettools.vscode-dotnet-runtime-2.0.5/dist/extension.js:2:748180)
	at Pt.request (/Users/ruimarinho/.vscode/extensions/ms-dotnettools.vscode-dotnet-runtime-2.0.5/dist/extension.js:2:748273)
	at process.processTicksAndRejections (node:internal/process/task_queues:95:5)

Error
Failed to download .NET 8.0.5~arm64:
No internet connection detected: Cannot install .NET

Error
Failed to download .NET 8.0:
.NET Acquisition Failed: Installation failed: Error: No internet connection detected: Cannot install .NET
```

### Reproduction Steps

Clone .net maui repo, make sure to make maui workloads installed

from the command line do:

checkout the net9 branch
donet tool restore
dotnet cake --targets=VSCODE

### Expected Behavior

Should not throw error.

### .NET Install Tool Version

v2.0.5

### Your operating system? Please provide more detail, such as the version and or distro above.

OSX / Darwin / Mac

###  Versions
```
 ~ dotnet --list-sdks
6.0.420 [/usr/local/share/dotnet/sdk]
6.0.422 [/usr/local/share/dotnet/sdk]
7.0.314 [/usr/local/share/dotnet/sdk]
7.0.315 [/usr/local/share/dotnet/sdk]
7.0.316 [/usr/local/share/dotnet/sdk]
8.0.203 [/usr/local/share/dotnet/sdk]
9.0.100-preview.4.24267.66 [/usr/local/share/dotnet/sdk]
```

**Comment by nagilson** [comment]
Thank you, this is an issue with the axios cache interceptor web request library. We got an update from them and hope it will be fixed when we release their patch with 2.0.6.

**Comment by nagilson** [comment]
Are you still seeing this in 2.0.6? We think it's fixed.
